Automattic/wc-calypso-bridge
# Remove email over-rides

Fixes #25 

Now that 3.3 is out the door, we can leverage the new `woocommerce_get_edit_order_url` filter to set admin order URLs to wordpress.com instead of over-riding the email templates... which become out of date and lead to support requests.

__To Test__
- Upload this branch as a plugin to a test site
- Place an order on the test site
- Open up the email sent to the address configured to receive new order emails within Woo, and verify the the order link target is for wordpress.com, not woo on the remote site

![order-link](https://user-images.githubusercontent.com/22080/36126027-7f3587b4-100b-11e8-9d7b-96e1b1795bfb.png)

__Random Musings__
We still don't have a way to determine if an AT site is primarily managed on WordPress.com vs wp-admin - but if/when we do, this filter logic could return the default wp-admin order edit url instead of the wpcom one for sites that are managed via wp-admin.
